.github/workflows/release-rs.yml

```
@@ -24,11 +24,12 @@ jobs:
     steps:
       # Generating a GitHub token, so that PRs and tags created by
       # the release-plz-action can trigger actions workflows.
+      # `app-id` is deprecated in favor of `client-id`.
       - name: Generate GitHub token
         uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)